packages/react-components/src/components/abstract-output-component.tsx

```diff
@@ -94,7 +100,11 @@ export abstract class AbstractOutputComponent<P extends AbstractOutputProps, S e
                 {outputName}
                 <i id={this.props.traceId + this.props.outputDescriptor.id + 'handleSpinner'} className='fa fa-refresh fa-spin'
                     style={{ marginTop: '5px', visibility: 'hidden'}} />
+                {this.state.optionsDropdownOpen !== undefined && <button title='Show View Options' className='remove-component-button' onClick={this.openOptionsMenu}>
+                    <FontAwesomeIcon icon={faBars} />
```

Comment:
I think the options menu button should be snapped below the close button, not floating when output height is resized.